common/rakelib/changelog.rake

```diff
@@ -65,6 +68,6 @@ namespace :changelog do
     body = Utils.top_changelog_entry
 
     repo_name = File.basename(`git remote get-url origin`.chomp, '.git').freeze
-    client.create_release("SwiftGen/#{repo_name}", tag, :body => body)
+    client.create_release("SwiftGen/#{repo_name}", tag, body: body)
```

Comment:
🤔 does that with with system ruby? I though that syntax was Ruby 2.1 or 2.2 only?